### PR TITLE
Add ci build

### PR DESCRIPTION
**Pull request by aureamunoz**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: Primaza Build
+
+on:
+  push:
+    branches: [ master ]
```
Comment (cmoulliard) [inline .github/workflows/build.yml:5]: Should we use main instead of master ?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+#jobs:
+#  build-jdk11:
+#    name: "JDK 11 Build"
+#    runs-on: ubuntu-latest
+#
+#    steps:
+#      - uses: actions/checkout@v2
+#      - name: Set up JDK 17
```
Comment (cmoulliard) [inline .github/workflows/build.yml:17]: Are you installing jdk 17 or 11 ?

File: .github/workflows/build.yml (hunk 1 above)
Comment (aureamunoz) [reply .github/workflows/build.yml:5]: oh, absolutely

File: .github/workflows/build.yml (hunk 2 above)
Comment (aureamunoz) [reply .github/workflows/build.yml:17]: oups this is copy/paste remainings

**Comment by aureamunoz** [comment]
fixed, wdyt?

**Comment by cmoulliard** [comment]
LGTM. Can be merged

**Pull request merged**